index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
+        <nav>
+            <a href="">
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
+    </header>
+
+    <main>
+        <section class="Profile">
+                <h1>I’m Temesgen <br> Glad to see you!</h1>
+                <p>
+                    I’m a software developer! I can help you
+                    build a product, feature or website. Look
+                    through some of my work and experirience!
+                    If you like what you see and have a project
+                    you need coded, don't hestitate to contact me.
+                </p>
+            <h4>Lets connect</h4>
```

Comment:
As required by the previous reviewer, please update the design to stick to Figma.

Please update the size and the color ant put the text in uppercase
Your design:

<img width="168" alt="image" src="https://user-images.githubusercontent.com/7243132/170825597-a7bd25ae-5dbd-4896-8c82-aeec096d969e.png">

Figma
<img width="217" alt="image" src="https://user-images.githubusercontent.com/7243132/170825623-fad82682-a712-42d4-adc8-bcf07c17ee51.png">